### ISSUE TITLE
Complete status icons

### ISSUE THREAD

**Issue by Kriesse** [datproject/dat-desktop#184]
They're confusing right now, need redesign. The current icons do not represent the possible states.

## Download Status:

- downloading ("in progress")
- downloading complete
- not downloading (because not connected to network)
- not downloading (because paused by user)

## Upload Status:

- uploading ("in progress")
- not uploading (because not connected to network)
- not uploading (because paused by user)

## Upload/Download Combination Status:

- downloading + uploading
- downloading + not uploading (because uploading paused)
- downloading complete + uploading
- downloading complete + not uploading (because uploading paused)
- not downloading + not uploading (because both paused)
- not downloading + not uploading (because not connected to network)

**Comment by Kriesse** [comment]
As shown in the suggestion, I think we should tie this in with another feature: Users should be able to pause the transfer (download+upload) of a dat. In context:

![image](https://cloud.githubusercontent.com/assets/53684/23162905/83d1fa1c-f830-11e6-8022-202791ddbae1.png)

**Comment by juliangruber** [comment]
moved pause/resume to a new issue: https://github.com/datproject/dat-desktop/issues/266

**Comment by Kriesse** [comment]
added the new icons that are needed. this is now blocked by #266.

**Comment by yoshuawuyts** [comment]
Closing in favor of https://github.com/datproject/dat-desktop/issues/266
